test/client/LedgerClient.spec.ts

```diff
@@ -323,4 +323,10 @@ describe("DamlLedgerClient", () => {
         assert(spy.calledOnceWithExactly(ledgerId));
     });
 
+    it('should correct list known parties', async ()=>{
```

Comment:
These tests are there to ensure the integration of the individual service client in the bundled `LedgerClient` implementation. A simple check that requires no mocking as in other test is sufficient.